docs/modules/sql/pages/functions-and-operators.adoc

```diff
@@ -395,6 +398,11 @@ This logic causes that any time value between years 1971 up to year 2968 is conv
 |Converts `TIMESTAMP_WITH_TIMEZONE` values to an `EPOCH` value in milliseconds.
 |`SELECT TO_EPOCH_MILLIS(date'2022-02-22')`
 |1645484400000
+
+|`TO_CHAR(datetime, format_mask [, nls_language])`
```

Comment:
@burakgok didn't we have docs for `TO_CHAR` function?

Reply:
@k-jamroz It was planned to have a dedicated section or page since its format differs from Postgres and Oracle.
@Serdaro Can you reopen #703?